addons/account/models/partner.py

```diff
@@ -876,3 +876,8 @@ def _merge_method(self, destination, source):
         if self.env['account.move.line'].sudo().search([('move_id.inalterable_hash', '!=', False), ('partner_id', 'in', source.ids)], limit=1):
             raise UserError(_('Partners that are used in hashed entries cannot be merged.'))
         return super()._merge_method(destination, source)
+
+    def _deduce_country_code(self):
+        # to override
+        self.ensure_one()
+        return self.country_code
```

Comment:
```suggestion
    def _deduce_country_code(self):
        """ overrides account
        Deduce the country code based on the information available.
        we have three cases:
        - country_code is BE but the VAT number starts with FR, the country code is FR, not BE
        - if a country-specific field is set (e.g. the codice_fiscale), that country is used for the country code
        - if the VAT number has no ISO country code, use the country_code in that case.
        """
        self.ensure_one()

        country_code = self.country_code
        if self.vat and self.vat[:2].isalpha():
            country_code = self.vat[:2].upper()
        return country_code
```
Why "to override" ?
Also, you absolutely need to put the whole method in `account` since you are overridding it in l10n_fr/it/no etc that don't depend of ubl_cii

Reply:
but we don't want to deduce the country code in l10n_be if ubl module is not installed :thinking: